composer.json

```diff
@@ -16,7 +16,7 @@
         "knplabs/knp-paginator-bundle": "^6.1",
         "mpdf/mpdf": "^8.1.2",
         "openspout/openspout": "^4.23",
-        "phpoffice/phpspreadsheet": "^1.25",
+        "phpoffice/phpspreadsheet": "^1.29",
```

Comment:
These changes to composer.json should not be necessary, because ^ ensures that minor versions are updated 

See
https://getcomposer.org/doc/articles/versions.md#summary

~ Tilde tælopper terminelle tal.
^ Circumflex sikrer spæde skridt.